src/App.js

```diff
@@ -2,9 +2,9 @@ import 'react-native-gesture-handler';
 import React from 'react';
 import { View, StyleSheet } from 'react-native';
 import { Provider as SosProvider } from 'state/SosContext';
```

Comment:
u can move this to index and then import { AuthProvider , LocationProvider, SosProvider } from 'state/';